tutorials/shaders/compute_shaders.rst

```diff
@@ -0,0 +1,242 @@
+.. _doc_compute_shaders:
+
+Using ComputeShaders
+====================
```

Comment:
```suggestion
Using compute shaders
=====================
```
ComputeShader is not a type, so this should use the two words separated a lowercase